lib/tasks/attendance_csv_importer.rake

```diff
@@ -4,7 +4,9 @@ task attendance_csv_importer: :environment do
   # Pull single file from Wonderschool URL
   # Process single file through Attendance Processor
   # Archive file to S3
-  AttendanceCsvImporter.new.call
+  start_date = ENV['start_date']&.to_date || 1.year.before
+  end_date = ENV['end_date']&.to_date || 0.days.after
+  AttendanceCsvImporter.new.call(start_date, end_date)
```

Comment:
Don't forget to update this implementation to initialize with the named variables!